packages/shared/src/focus-visible.ts

```diff
@@ -114,20 +114,26 @@ export const FocusVisiblePolyfillMixin = <
         // document:
         connectedCallback(): void {
             super.connectedCallback && super.connectedCallback();
-            if (this[$endPolyfillCoordination] == null) {
-                this[$endPolyfillCoordination] = coordinateWithPolyfill(this);
-            }
+            requestAnimationFrame(() => {
+                if (this[$endPolyfillCoordination] == null) {
+                    this[$endPolyfillCoordination] = coordinateWithPolyfill(
+                        this
+                    );
```

Comment:
guessing you'll want to collapse 120/121 back into line 119.

Reply:
I don't do that... Prettier owns all the new lines 🤷‍♂️  it gets tabbed over one more time in #984, so not sure it's worth messing with?